### PR TITLE
Starts #1873 - Adding Islandora Cookbook to main docs

### PR DESCRIPTION
**Pull request by manez**
## Purpose / why

Updates the "extending Islandora" page to be mostly cookbook, adds first recipe.  Starts work on #1873 

## What changes were made?

Edited a page, added a new screenshot, and added a new recipes page and folder. There are four other receipts that will need to come over, but I figured it was better to get a proof of concept out there sooner rather than wait until I (or anyone else who wants to help!) get them all converted. 

Did some minor de-8ing as well.

## Verification

Review for content. Ideally do a test build because new pages are being added.

## Interested Parties

@Islandora/documentation

---

## Checklist

> __Pull-request reviewer__ should ensure the following

* [ ] Does this PR link to related [issues](https://github.com/Islandora/documentation/issues/)?
* [ ] Does the proposed documentation align with the [Islandora Documentation Style Guide](https://islandora.github.io/documentation/contributing/docs_style_guide/)?
* [ ] Are the changes accurate, useful, free of typos, etc?
* [ ] Does this PR update the _last updated on_ date on the documentation page?

> __Person merging__ should ensure the following
* [ ] Does mkdocs still build successfully? (This is indicated by TravisCI passing. To test locally, and see warnings, see [How To Build Documentation](https://islandora.github.io/documentation/technical-documentation/docs-build/).)
* [ ] If pages are renamed or removed, have all internal links to those pages been fixed?
* [ ] If pages are added, have they been linked to or placed in the menu?
* [ ] Did the PR receive at least one approval from a committer, and all issues raised have been addressed?

### CODE REVIEW

File: docs/user-documentation/recipes/alexa_search.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Accessing Islandora with Amazon Alexa
+The following recipe details how to connect Islandora with Alexa, using custom Alexa skills and the Drupal Alexa module. The potential applications are broad:
+- Send Citations, metadata, whatever we want to the user if they have set up their email
```
Comment (rosiel) [inline docs/user-documentation/recipes/alexa_search.md:4]: This list needs a new line before it, it's rendering as part of the paragraph

File: docs/user-documentation/recipes/alexa_search.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+     1. Log in https://developer.amazon.com go to Alexa > Skills Kit > add new skill
+     1. Pick an _Invocation Name_ 
+     1. Configure SSL etc (Tutorial: https://www.drupal.org/docs/8/modules/alexa/tutorial) 
+     1. Save config, write down the Application ID on the Skill Information Tab!
+1. Integrate with Drupal
+     1. Install the Alexa module using Composer:  `composer require "drupal/alexa"`
+     1. Enable modules (Alexa, Alexa_Demo) 
+     1. Go to Config > Alexa Configuration and put in the Application ID. 
+1. Test with one question and answer (“Clawbster, say Hello world?” “Hello World!”) Now you can add as many questions and answers as you can configure:
+![Screenshot of google form with questions and answers for chatbot](../docs/assets/recipe_alexa.png)
```
Comment (rosiel) [inline docs/user-documentation/recipes/alexa_search.md]: link to image not working, directing to 'user-documentation/docs/assets/recipe_alexa.png'

**Review by rosiel — COMMENTED** [review]
In general, awesome!

A couple formatting/linking things.

File: docs/user-documentation/recipes/alexa_search.md (hunk 2 above)
Comment (manez) [reply docs/user-documentation/recipes/alexa_search.md]: Bother! I wrestled with that thing through a few iterations and I thought I had it working in preview. I don't know why links are such strange magic for me.

File: docs/user-documentation/recipes/alexa_search.md (hunk 1 above)
Comment (manez) [reply docs/user-documentation/recipes/alexa_search.md:4]: On it!

File: docs/user-documentation/recipes/alexa_search.md (hunk 2 above)
Comment (rosiel) [reply docs/user-documentation/recipes/alexa_search.md]: I hear ya!

`Screenshot of google form with questions and answers for chatbot `
worked for me.

File: docs/user-documentation/extending.md [hunk 3]
```diff
@@ -1,13 +1,121 @@
...
  
 Islandora 8 is also compatible with most Drupal themes. 
+
+## Introduction
+
+In the spirit of [Islandora Awesome](https://github.com/Islandora-Labs/islandora_awesome), Islandora Cookbook is a curated list of great modules and other tools for Islandora 8. Because Islandora 8 is more tightly integrated with Drupal, most of these 'recipes' require only Drupal modules as ingredients.
+
+We offer this list for discovery, but do not officially provide support for any of these modules. 
+
+# Recipes
```
Comment (eldonquijote) [inline docs/user-documentation/extending.md]: `## Recipes`

(the table of contents for the page won't render if the headings aren't properly nested)

File: docs/user-documentation/extending.md [hunk 4]
```diff
@@ -1,13 +1,121 @@
...
+
+- [Amazon Alexa](recipes/alexa_search.md) - This recipe explains how to access Islandora with Amazon Alexa.
+- [Collection Searching] - 
+This recipe provides instructions how to configure a shallow or deep search in a specific collection.
+- [Date Range Slider Facet] - This recipe explains how to include a facet for date field(s) that presents itself as a range slider.
+- [Exhibitions and TimeLine JS] - This recipe outlines how to build exhibits in Islandora 8, including the deployment of TimeLineJS.
+- [Digital Preservation] - A detailed overview of how to approach digital preservation in Islandora 8. Includes features that are currently avaiable, both within Islandora and by using compatible tools.
+
+# Ingredients
```
Comment (eldonquijote) [inline docs/user-documentation/extending.md]: From here down, until `## Other Resources`, all headings need an additional #

File: docs/user-documentation/recipes/alexa_search.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+     1. Configure SSL etc (Tutorial: https://www.drupal.org/docs/8/modules/alexa/tutorial) 
+     1. Save config, write down the Application ID on the Skill Information Tab!
+1. Integrate with Drupal
+     1. Install the Alexa module using Composer:  `composer require "drupal/alexa"`
+     1. Enable modules (Alexa, Alexa_Demo) 
+     1. Go to Config > Alexa Configuration and put in the Application ID. 
+1. Test with one question and answer (“Clawbster, say Hello world?” “Hello World!”) Now you can add as many questions and answers as you can configure:
+![Screenshot of google form with questions and answers for chatbot](/docs/assets/recipe_alexa.png)
+
+## Notes
```
Comment (eldonquijote) [inline docs/user-documentation/recipes/alexa_search.md]: Suggestion: put this at the beginning in an admonition:
`!!! note`
 `   This recipe has not been extensively tested.`

**Comment by manez** [comment]
I think I've addressed everything @rosiel @eldonquijote

**Pull request merged**